app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue

```diff
@@ -145,6 +148,16 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 export default {
   data() {
     return {
+      systemAppNames: [
```

Comment:
Why do you have appName here ? 
How it will work if a configuration change one of this name ? or add a new one ?

Reply:
Yes you are right i will fix it.

Reply:
what will you do to fix ?

Reply:
Fixed